connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -1071,7 +1071,7 @@ private void awaitStopTask(ConnectorTaskId taskId, long timeout) {
             }
 
             if (!task.awaitStop(timeout)) {
-                log.error("Graceful stop of task {} failed.", task.id());
+                log.error(" Graceful stop of task {} failed.", task.id());
```

Comment:
nit: did you mean to add this space?

Reply:
Nope, that was definitely inadvertent so I will fix that.